docs/styleguide.md

```diff
@@ -0,0 +1,43 @@
+# Styleguide
+
+We follow the [Vue Styleguide](https://vuejs.org/v2/style-guide/) recommandation.
+
+Ui design configuration can be found in [tailwind.config.js](../tailwind.config.js).
+
+## Base
+
+We are using base pixel for body to `16px`
+
+## Grid
+
+- [Tailwind grid](https://tailwindcss.com/docs/grid-column/)
+- [tailwdin gap doc](https://tailwindcss.com/docs/gap)
+
+### Desktop
+
+- gap/margin: `24px - 1.5rem`
```

Comment:
I've put back ticks around values, and put them in list.
@f3ltron can you review this file? I'm not sure what you wanted to document here.

Reply:
I wanted to explain margin padding base pixel decision

Same for grids

Reply:
for the grid the thing is i never really use it but i think it should be great to use it instead of just flex w-1/3 etc. What do you think about?  @aurelienroux  too ?